libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -548,6 +558,16 @@ public static (IReadOnlyList<object>, string error) EvaluateChildren(Expression
 
         // Apply -- these are helpers for adding functions to the expression library.
 
+        public static string EvaluateLocale(IMemory state)
```

Comment:
Public method. Public class. Comments required.

Reply:
changed to private